test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpTraceExporterTests.cs

```diff
@@ -389,98 +389,6 @@ public void GetMetadataFromHeadersThrowsExceptionOnOnvalidFormat(string headers)
             throw new XunitException("GetMetadataFromHeaders did not throw an exception for invalid input headers");
         }
 
-        private static void AssertOtlpAttributes(
```

Comment:
I moved this method to a helper class since it will be useful for traces, metrics and logs.